assets/thanos-querier/deployment.yaml

```diff
@@ -33,18 +33,14 @@ spec:
             command:
             - sh
             - -c
-            - if [ -x "$(command -v curl)" ]; then curl http://localhost:9090/-/healthy;
-              elif [ -x "$(command -v wget)" ]; then wget --quiet --tries=1 --spider
-              http://localhost:9090/-/healthy; else exit 1; fi
+            - if [ -x "$(command -v curl)" ]; then curl http://localhost:9090/-/healthy; elif [ -x "$(command -v wget)" ]; then wget --quiet --tries=1 --spider http://localhost:9090/-/healthy; else exit 1; fi
```

Comment:
hmm, these changes are strange, but as far as i see `jb` just undid the newlines? :man_shrugging:

Reply:
I believe this is rather something in gojsontoyaml. We generate json, which doesn't have a concept of line length and then convert it to yaml. Either way I think we can live with it :)